haunt-roll-fail/arcs/game-common.scala

```diff
@@ -707,7 +708,8 @@ object CommonExpansion extends Expansion {
 
             f.log("taxed", c, "in", r, x)
 
-            f.taxed :+= c
+            if ((f.lores.has(WarlordsCruelty) && game.declared.contains(Warlord)).not)
```

Comment:
This would work right most of the time.
But for a very rare *tax* -> *discard* -> *tax again* sequence.
Oh, and also *tax* -> *secure* Populist Demands *declare* Warlord -> *can't tax*.